ctapipe/instrument/camera/geometry.py

```diff
@@ -371,7 +419,7 @@ def to_table(self):
             [self.pix_id, self.pix_x, self.pix_y, self.pix_area],
             names=["pix_id", "pix_x", "pix_y", "pix_area"],
             meta=dict(
-                PIX_TYPE=self.pix_type,
+                PIX_TYPE=self.pix_type.value,
```

Comment:
again, this is not very readable the user has to know what pixel type this integer corresponds to - why not just use the string representation and then it doesn't break the data model?

Reply:
This is an `Enum` with str values compatible to the ones we had before. Not an `IntEnum`.

Reply:
ah, I thought they were integer values. Sorry, it does what I suggested then.